tests/test_efadv.py

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: (GPL-2.0 OR Linux-OpenIB)
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All rights reserved.
+"""
+Test module for efa direct-verbs.
+"""
+
+import errno
+from pyverbs.base import PyverbsRDMAError
+import pyverbs.providers.efa.efadv as efa
+from tests.base import PyverbsAPITestCase
+import unittest
+
+
+class EfaDvTest(PyverbsAPITestCase):
+    """
+    Test various functionalities of the direct verbs class.
+    """
+    def test_efadv_query(self):
+        """
+        Verify that it's possible to read EFA direct-verbs.
+        """
+        for ctx, attr, attr_ex in self.devices:
+            with efa.EfaContext(name=ctx.name) as efa_ctx:
```

Comment:
New tests are not supposed to be doign the 'for .. self.devices' loop, this should eventually get fixed up for the DV tests so that RDMATestCase can create the right context and skip if necessary.